helm/reana/templates/NOTES.txt

```diff
@@ -1,22 +1,20 @@
+The REANA system has been installed!
 
-The REANA system has been installed:
+If you are deploying REANA for the first time, there are a few steps left to
+finalise its configuration.
 
-If you are installing REANA for the first time, there are a few steps left to finalise its configuration.
+1. Initialise the database:
 
-1. Get the REANA-Server pod name:
+    $ kubectl exec deployment/reana-server -- ./scripts/create-database.sh
 
-    $ REANA_SERVER=$(kubectl get pod -l "app={{ .Release.Name }}-server" -o name -o jsonpath='{.items[0].metadata.name}')
+2. Create administrator user and corresponding access token:
 
-2. Initialise the database:
+    $ mytoken=$(kubectl exec deployment/reana-server -- flask reana-admin \
+        create-admin-user --email john.doe@example.org --password mypass1234)
 
-    $ kubectl exec $REANA_SERVER -- ./scripts/setup
+3. Store administrator access token as Kubernetes secret:
 
-3. Create your administrator user and store the token:
-
-    $ kubectl exec $REANA_SERVER -- flask reana-admin create-admin-user --email john.doe@example.org --password mysecretpassword
-    <reana-admin-access-token-value>
-    $ read -s REANA_ADMIN_ACCESS_TOKEN  # paste the secret here
     $ kubectl create secret generic {{ .Release.Name }}-admin-access-token \
-      --from-literal=ADMIN_ACCESS_TOKEN="$REANA_ADMIN_ACCESS_TOKEN"
+        --from-literal=ADMIN_ACCESS_TOKEN="$mytoken"
 
```

Comment:
Hmm, perhaps we should also add step 4 saying how to set tokens and run first example?

Reply:
Added.

Reply:
BTW while doing so I noticed that we are using `reana_url` for something that is not an "URL" strictly speaking; it is rather a "hostname", because the protocol part is left out... so it would be better to call it `reana_hostname` rather than `reana_url`. Let's discuss IRL whether we change this for clarity before releasing?